source/core/src/main/com/csse3200/game/areas/forest/ForestGameArea.java

```diff
@@ -527,31 +527,31 @@ private void spawnFriendlyNPCs() {
 
         // Cow
         generator = () -> NPCFactory.createCow(player, this.enemies);
-        spawnRandomNPC(generator, ForestSpawnConfig.NUM_COWS);
+        spawnRandomNPC(generator, ForestSpawnConfig.NUM_COWS,1);
 
         // Fish
         generator = () -> NPCFactory.createFish(player, this.enemies);
-        spawnRandomNPC(generator, ForestSpawnConfig.NUM_FISH);
+        spawnRandomNPC(generator, ForestSpawnConfig.NUM_FISH, 2);
 
         // Lion
         generator = () -> NPCFactory.createLion(player, this.enemies);
-        spawnRandomNPC(generator, ForestSpawnConfig.NUM_LIONS);
+        spawnRandomNPC(generator, ForestSpawnConfig.NUM_LIONS,1 );
 
         // Turtle
         generator = () -> NPCFactory.createTurtle(player, this.enemies);
-        spawnRandomNPC(generator, ForestSpawnConfig.NUM_TURTLES);
+        spawnRandomNPC(generator, ForestSpawnConfig.NUM_TURTLES, 2);
 
         // Eagle
         generator = () -> NPCFactory.createEagle(player, this.enemies);
-        spawnRandomNPC(generator, ForestSpawnConfig.NUM_EAGLES);
+        spawnRandomNPC(generator, ForestSpawnConfig.NUM_EAGLES, 2);
```

Comment:
Shouldn't this (and the magpie) be in zone 3?

Reply:
Thank you! I saw the bees were in zone 2 and assumed that was air, but this is now fixed.